docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -84,6 +84,22 @@ For more information, refer to {agent-issue}2066[#2066].
 Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
 ====
 
+[[known-issue-issue-2086]]
+.Beats started by agent can be configured without a valid output
```

Comment:
```suggestion
.Beats started by agent may fail with `output unit has no config` error
```
I think the original could be misread because the passive voice makes it unclear as to what is doing the configuring (user or system). I suggest including the error message because that's what users might search on.

Reply:
Thanks, this is much clearer